### DISCUSSION TITLE
【脑洞】奇年祭：奇忽数码物丰年祭+奇忽NFT数码物农贸市场

### DISCUSSION THREAD

**Discussion by Lupingyue** [qihuju/qihuju.github.io#39] [Ideas]
这个脑洞是 #37 讨论的成果「speakers： @MingxuanXie @askender @Tz-H @midai17 @MRchildNEO @Lupingyue 」

暂定：2022年秋天，举行首届“奇年祭”

“奇年祭”全称：奇忽数码物丰年祭+奇忽数码物NFT农贸市场

首届奇年祭小目标：108个Mozilla数码物房间

Mozilla数码物房间的定义，由 @Tz-H 梳理：

1. 通过Clubhouse的话题活动收集沉淀数码物，并以每一期话题房间的形式加以陈列展示，数码物的所有者归属于在话题房间中的提出该数码物（话题）的讲者

2. 为讲者提供时空穿梭小屋，小屋提供“虫洞”可以穿越到以往的不同话题房间，虫洞的终点（展示品）是讲者在那一期话题房间所沉淀展示的数码物

3. 参观者即可以依托于话题穿梭于不同的话题房间，也可以依托于讲者，通过讲者的时空穿梭小屋去浏览体验讲者在不同话题下所沉淀下来的数码物

**Comment by Lupingyue** [comment]
> 这个脑洞是 #37 讨论的成果「speakers： @MingxuanXie @askender @Tz-H @midai17 @MRchildNEO @Lupingyue 」
> 
> 
> 
> 暂定：2022年秋天，举行首届“奇年祭”
> 
> 
> 
> “奇年祭”全称：奇忽数码物丰年祭+奇忽数码物NFT农贸市场

@Sleeper2060 和我在discord的讨论：

@Sleeper2060 
这个丰年祭的idea还蛮新颖好玩的，不过好像还欠了点互动。虚拟展厅现在有点变成了一个储物的空间。或者可不可以就好像寺庙那样，来的人可以点香或许愿。这个需要再详细的策划和内容的探讨。

@Lupingyue 
1，所有的数码物都可以做成nft， @MingxuanXie  的桃花源赛博展厅就是一个nft画廊
2，所有的数码物都可以嵌入 @askender 的like coin
3，加上你说的点香和许愿，差不多就完整了

**Comment by Lupingyue** [comment]
奇年祭的构想，抛砖引玉，请大家拍砖：

**主题：重新思考摩尔定律**

**——奇年祭是周期性的送别，也是周期性的迎接**

奇忽局的名称针对“奇点临近”的摩尔定律福音教派，特别是以“奇点预言家”库兹韦尔发布的确定性福音书《奇点临近》，以及在实践上相呼应的马斯克的“火星教派”，虽然马斯克以及生前的霍金经常发表AI杀人的警示，但是马斯克无疑是AI奇点教的最大受益者。
然而，灰犀牛和黑天鹅横飞的这几年「疫情、川普、脱欧等」却上演的是“不确定性”的戏码。
但是台积电、三星为代表的“纳米”战场依然是当前的奇点主战场，与此同时，有人意识到“摩尔定律”正在走向失效，除非量子计算机获得突破。
首届奇年祭关注这个当下最紧迫的人类命运问题：摩尔定律会失效吗？摩尔定律如果失效人类将会走向何方？火星「向外」？还是内在的虚拟现实的元宇宙世界「向内」？

**五大板块**

> 第一板块：摩尔定律失效

地球大脑「多年前IBM的营销术语」及其替代方案：火星大脑or月球大脑

> 第二板块：多巴胺的崛起

灰犀牛、黑天鹅横行的21世纪20年代，机器大脑面临摩尔定律瓶颈，人脑开始在元宇宙「虚拟现实」中寻求麻醉、享乐、高潮、巅峰体验，肉脑的复兴正在填补人工智能大脑混乱所带来的各种真空地带：政治、经济、社会和文化的各种碎片化的真空地带

> 第三板块：人机双修

机器想要什么？这是KK的发问。目前我们看到的是：人想要机器模仿人，机器人也正在按照这个目标继续拟人化：机器视觉、机器听觉、机器空间识别……都没有逃脱人的眼耳鼻舌身意框定的赛道。
人类也因为植入了电子器官「智能手机」开始进入“量化自我”，跟机器签订浮士德协议，规模化yesorno模式的app契约常态化。

> 第四板块：赛博格

如果人机双修只是一种“readymade cyborg”，现成的赛博格，那么人机接口、战锤模式、赛博朋克2020模式的身体改造则是很快就会降临的现实

> 第五板块：缸中大脑

汇聚以上四个板块，20世纪20年代的“维特鲁威人”是什么样的？会是《西部世界》里那款“维特鲁威人”吗？最关键的是，肉脑和机器大脑相互融合成为一种什么样的双脑？我们如何呈现人类这个时期的“自画像”？


**奇年祭的呈现模块：Mozilla的108个房间构成的赛博游牧部落**

奇忽局club在呈现形式上有：

> clubhouse开放讨论

> clubhouse讨论脑图「第五期开始」

> clubhouse讨论脑图+Mozilla超链接文献展房间「第十四期开始」

那么如果第15期到第108期，都可以实现第14期的呈现形式，就可以构成一个108个相互链接的Mozilla房间，这就是一个简单版本的赛博游牧部落的雏型

一旦有了这样一个可以预期的“赛博游牧部落”，年度的部落嘉年华就需要提上日程，这个嘉年华就是年度“奇年祭”


**首届奇年祭时间：2022年9月-12月**

2022年是21世纪20年代第二个摩尔定律的结束时间点「按照18个月一个摩尔定律周期计算」，也可能是最后的摩尔定律有效期
首届奇年祭重新思考摩尔定律带来的奇点福音，也有可能是对摩尔定律的一次送终
之后，是更加激进的加速「量子计算」，还是陷入减速，不得而知

**——奇年祭是周期性的送别，也是周期性的迎接**

**Comment by askender** [comment]
支持。但我是冷水党，简单拍砖。 几个板块：
1. 芯片领域，我对翻倍没有强追求，量子计算也是观望，这个领域还是靠arm amd对抗没落的intel , 以及risc-v的兴起，貌似在这个领域我的贡献就是多买机器 。至于向外探索，也只能买股票支持一下，以及科普(我有个在造火箭的朋友) 
2.  是我最关心的。精神领域和肤浅娱乐的对抗，人类解决基础需求之后的痛苦来源
3.  vr&ai 也是dweb的反抗之路，预计dweb几十年都难以超过20%， web2的AI不是好AI， vr我也不希望被web2引导
4. 脑机接口领域和生物学领域 后者不太关注改造而是新的不平等，前者预计对人类精神领域没有大的触动，而没有dweb的脑机也不是好的脑机
5. 是科幻和哲学领域。   是指导2和3的东西

如果是谈论和幻想倒是都可以，如果是实干， 2 3容易有作品。       奇点本身也是技术乐观派的幻想，实干派同时是乐观者和悲观者，希望将一件不可能的事情做成。  2 依靠静下心来创作    3 是在技术上有更高的追求，达到sota (论文领域的常见词， state-of-the-art)

如果以奇忽局的几个关键词来看：  奇忽∈5 科幻∈5    遊戲∈2  藝術∈2    區塊鏈∈3  （vr-思维导图-wiki-知识图谱∈3  AI∈3 ）      1(物理基础)和4（生物改造）比较硬核，容易被做成5，但实际也是技术类的3

**Reply by Lupingyue** [reply]
@askender 序号排错了：2，3，4应该是3，4，5，因为2乱入1了😄

**Reply by askender** [reply]
已修正。 2后面忘了加换行，因为此前是在discord编辑的，没注意到这个问题，

**Reply by Lupingyue** [reply]
> 2. 是我最关心的。精神领域和肤浅娱乐的对抗，人类解决基础需求之后的痛苦来源

所以你对目前的“肤浅娱乐”的web2的现状很不满？你觉得这是“中心化”平台的商业模式的必然结果吗？除了web3，不可能在web2现有的框架下解决“肤浅娱乐”的问题吗？我觉得这已经不是一个技术判断，而是一个技术伦理甚至是政治判断了

> 3. vr&ai 也是dweb的反抗之路，预计dweb几十年都难以超过20%， web2的AI不是好AI， vr我也不希望被web2引导

这个20%的说法让我想到马斯克的特斯拉的一个目标是：新能源要提升到50%，以此来迫使就能源下降到50%

> 4. 脑机接口领域和生物学领域 后者不太关注改造而是新的不平等，前者预计对人类精神领域没有大的触动，而没有dweb的脑机也不是好的脑机

没有dweb的脑机不是好的脑机，又是一个技术伦理问题。我发现你对技术伦理的关注似乎超过对技术本身的关注

> ……3 是在技术上有更高的追求，达到sota (论文领域的常见词， state-of-the-art)

state-of-the-art，这个第一次听说，直觉很靠谱
